resources/language/resource.language.de_de/strings.po

```diff
@@ -708,7 +708,7 @@ msgstr "Es gab ein Problem bei der Anmeldung, möglicherweise ist Ihr Account ni
 
 msgctxt "#30181"
 msgid "Always show the subtitles when the preferred audio language is not available"
-msgstr ""
+msgstr "Untertitel immer anzeigen wenn bevorzugte Audiosprache nicht verfügbar ist"
```

Comment:
There is a comma missing after "anzeigen".

Reply:
Thanks. fixed!